frontend/projects/upgrade/src/app/features/dashboard/home/components/table-row/table-row.component.html

```diff
@@ -32,7 +32,6 @@
 
```

Comment:
why are linting changes committed for files that aren't relevant to this PR?

Reply:
I am not sure but these changes are due to merge conflicts we were getting right? @RidhamShah